zarr/storage.py

```diff
@@ -1308,6 +1318,8 @@ def __init__(self, url, normalize_keys=False, key_separator=None,
                  create=False,
                  missing_exceptions=None,
                  **storage_options):
+        if not self._fsspec_installed():
```

Comment:
```suggestion
        if not self._fsspec_installed():  # pragma: no cover
```

This should keep coverage at 100%